lib/dropbox_api/endpoints/rpc_notify.rb

```diff
@@ -3,7 +3,7 @@ module DropboxApi::Endpoints
   class RpcNotify < DropboxApi::Endpoints::Rpc
     def build_connection
       @connection = @builder.build('https://notify.dropboxapi.com') do |c|
-        c.headers.delete 'Authorization'
+        c.builder.handlers.delete(Faraday::Request::Authorization)
```

Comment:
This seems to be the documented way to remove existing handlers: https://www.rubydoc.info/gems/faraday/Faraday/RackBuilder#delete-instance_method